src/vellum/workflows/nodes/displayable/bases/utils.py

```diff
@@ -148,7 +181,12 @@ def process_additional_prompt_outputs(outputs: List[PromptOutput]) -> Tuple[str,
             try:
                 json_output = json.loads(output.value)
             except (json.JSONDecodeError, TypeError):
-                pass
+                stripped_value = _strip_markdown_json(output.value)
```

Comment:
We should be able to move this into the `try` block so that we don't have nested logic in the `except` block